### PR TITLE
Link: Update deprecated Bears documentation link

### PR DESCRIPTION
**Pull request by mamat-rahmat**
Old link pointing to https://github.com/coala/bear-docs that is
deprecated.

Fixes https://github.com/coala/documentation/issues/495

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -226,7 +226,7 @@ sequentially.
 .. note::
 
     For a list of configuration options for the bears, take a look at our
-    `bear-docs <https://github.com/coala/bear-docs>`_ documentation.
+    `Coala Bears <https://coala.io/#/languages>`_ documentation.
```
Comment (jayvdb) [inline Users/Tutorial.rst]: coala is always written with a lowercase c.

Comment (jayvdb) [reply Users/Tutorial.rst]: `... our <coala languages> directory.` ?

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/496/commits/7c192aec9762a77273156677351e550efb8f319c

File: Users/Tutorial.rst (hunk 1 above)
Comment (mamat-rahmat) [reply Users/Tutorial.rst]: done

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**